### ISSUE TITLE
[Bug] Panorama Toggle/Mixin Bug

### ISSUE THREAD

**Issue by eithli** [juancarloscp52/BedrockIfy#240]
A few minor bugs have been discovered, as with the new update to this mod. These bugs appear to be related to the panorama settings (mixin & toggle) you made with the update. Furthermore, based on my testing, the following occurs entirely at random each time you launch the game; so the results with the bug are not consistent.

**Something to keep in mind:** I AM using a resource pack that changes the menu panorama (Vanilla Tweaks retro background), so this bug is caused primarily by the texture change. So the bug is in how this mod handles the texture of the menu screens.

**Transparency Issues**: This update appears to have broken transparency. The Bedrockify loading screen is blacked-out instead of being transparent.

_Pictures of the said issue above:_
![image](https://user-images.githubusercontent.com/109933737/226207372-496ef319-05c7-4b29-9946-da63a6027186.png)

**Stretched Backgrounds**: Furthermore, the background of some menus appears to be stretched out and distorted. I only have one image of the problem, but it occurs on the following screens.
- Create-A-World Screen.
- Loading Screen.
- Settings Screen.

_Picture for the said issue above:_
![image](https://user-images.githubusercontent.com/109933737/226206494-d5780b0f-9611-4795-99f1-3f74eb1d3351.png)

**TLDR**: Transparency issue & stretched out and distorted backgrounds to screens. **I use a resource pack**, so keep that in mind.

**Comment by eithli** [comment]
In addition, I didn't realize it was a toggle for the panorama screens, so that's my mistake. Regardless, the same problem persists even with there being a toggle. 

This separate comment is also for a minor bug. When the panorama setting is disabled, the loading screen transparency is messed up.

This is what I mean: 
![image](https://user-images.githubusercontent.com/109933737/226205221-271e3cb4-3c6c-4c9a-afff-7dfa7c81eaf8.png)

It should look like this: 
![image](https://user-images.githubusercontent.com/109933737/226205193-f254acfe-233a-4d37-87aa-d3ddd57a11b8.png)(without the stretched out loading screen ofc, but you get the point)

**Comment by eithli** [comment]
Last Update: Let me summarize what I've discovered so far as my comments above became so jumbled and I want it to be more readable.

Panorama Mixin Enabled ONLY: When enabled the world screen/create-a-world & the loading screen(s) all get stretched out and distorted. Now keep in mind I do have a texture pack that changes the panorama of the main screen, so I don't know if that's something to keep in mind.

Pictures for reference about the above: 
![image](https://user-images.githubusercontent.com/109933737/226206133-8e73dfc8-6ecb-404f-966c-22e4a82ee10a.png)
![image](https://user-images.githubusercontent.com/109933737/226206135-607668dc-d54b-4d2d-98c5-2cba61bedf99.png)
![image](https://user-images.githubusercontent.com/109933737/226206318-b5f3be1f-512a-4eac-b3e1-eae550606ed5.png)

Panorama Toggle Enabled ONLY: When enabled, the problems that existed when the mixin was enabled appear to be fixed, though there are some bugs. One being is that transparency on menu screens appears to be broken entirely; for example, the Bedrockify tooltip loading screen is completely black with no transparency. Another being is that the new create a world screen transparency near the top is also blacked out. It's not a major issue, but it is a bug.

Pictures for reference about the above:
![image](https://user-images.githubusercontent.com/109933737/226206196-2878ded0-b481-4115-9f40-68e6416d3376.png)
![image](https://user-images.githubusercontent.com/109933737/226206207-70d9355f-0940-4662-81b4-076d7bec287c.png)

With BOTH Being Enabled: When both are enabled, it appears that the glitch from when the mixin was enabled is only visible. Although the screens remain stretched and distorted, the transparency is no longer broken.

Pictures for reference about the above:
![image](https://user-images.githubusercontent.com/109933737/226206435-f78cc687-d69a-4b80-aa68-855f0e211727.png)
![image](https://user-images.githubusercontent.com/109933737/226206441-33c98677-598f-443c-9aeb-ac096ddf002e.png)
![image](https://user-images.githubusercontent.com/109933737/226206494-d5780b0f-9611-4795-99f1-3f74eb1d3351.png)

I hope that helps, and please excuse my writing in the photos above because I'm not very good at writing words with a mouse lol. I hope this helps, and have a great day!

**Comment by juancarloscp52** [comment]
Thank you so much for opening this issue. Both bugs should be fixed now and will be available in the next bedrockify version.
